pkg/controller/controller.go

```diff
@@ -1369,6 +1369,18 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) error {
 	return nil
 }
 
+// isPlanBindable returns whether the given ServiceClass and ServicePlan
+// combination is bindable.  Plans may override the service-level bindable
+// attribute, so if the plan provides a value, return that value.  Otherwise,
+// return the Bindable field of the ServiceClass.
+func isPlanBindable(serviceClass *v1alpha1.ServiceClass, plan *v1alpha1.ServicePlan) bool {
```

Comment:
This needs a warning somehow. It's not doing anything complicated, but in theory the serviceplan must 'belong to' the serviceclass. This function does not check that relationship.

My gut says this should be a method on service class that takes a plan, but I'm not sure that's worth it.

I don't think there is any action here.